src/store/channels-list/saga.test.ts

```diff
@@ -387,3 +287,12 @@ describe(fetchConversations, () => {
     ]);
   });
 });
+
+function subject(sagaFunction: any, args?: any, additionalProviders = []) {
+  return expectSaga(sagaFunction, args).provide([
+    ...additionalProviders,
+    [matchers.call.fn(chat.get), chatClient],
+    [matchers.call.fn(chatClient.getChannels), MOCK_CHANNELS],
+    [matchers.call.fn(chatClient.getConversations), MOCK_CONVERSATIONS],
+  ]);
+}
```

Comment:
```suggestion
function subject(...args: Parameters<typeof expectSaga>) {
  return expectSaga(sagaFunction, args)
    .provide([
      [matchers.call.fn(chat.get), chatClient],
      [matchers.call.fn(chatClient.getChannels), MOCK_CHANNELS],
      [matchers.call.fn(chatClient.getConversations), MOCK_CONVERSATIONS],
   ]);
}
```

See comment above. Don't take `additionalProviders` here as it's not necessary. Also, updated the function signature so typescript can help the tests get the arguments right.